src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -134,6 +123,10 @@ private void OnNodeAutoCompleteSearchControlVisibilityChanged(object sender, Dep
             // When launching this control, always start with clear search term.
             SearchTextBox.Clear();
 
+            Analytics.TrackEvent(
```

Comment:
Doesn't this get called twice - once when the control becomes visible and once when it gets hidden?

Reply:
`if (!(bool)e.NewValue)`
this takes care of those cases when control hides.

Reply:
@aparajit-pratap See line 120